app/models/fields_definition.rb

```diff
@@ -52,14 +52,13 @@ def self.editable_by(user)
 
   alias_attribute :position, :pos
 
-  # this is tested in unit/account_test
-  def self.create_defaults(account)
+  # This smells of :reek:NestedIterators: FieldsDefinition#self.create_defaults! contains iterators nested 2 deep
+  def self.create_defaults!(account)
     targets.each do |target|
       klass = target.constantize
-      klass.required_fields.each do |f|
-        label = DEFAULT_LABELS[f] || f.humanize
-        account.fields_definitions.create!({ :target => target, :name => f,
-                                             :label => label, :required => true })
+      klass.required_fields.each do |field|
```

Comment:
couldn't this logic be moved to `lib/fields/fields.rb`? this way I think it will not smells :reek:NestedIterators:. WDYT?

Reply:
I really don't want to touch this class or method at all for this PR... it is already a risky PR like to make it more risky... I only changed the `!` at the end of the method because it is what it is, and CodeClimate started complaining about things that were already wrong in this code... 🤷‍♀

Reply:
👍